jmeter/src/main/java/org/eclipse/hono/jmeter/HonoReceiverSampler.java

```diff
@@ -33,34 +33,62 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(HonoReceiverSampler.class);
 
-    private static final String USE_SENDER_TIME = "useSenderTime";
-    private static final String PREFETCH = "prefetch";
+    private static final String USE_SENDER_TIME                   = "useSenderTime";
+    private static final String SENDER_TIME_IN_PAYLOAD            = "senderTimeInPayload";
+    private static final String SENDER_TIME_VARIABLE_NAME         = "senderTimeVariableName";
+    private static final String RECONNECT_ATTEMPTS                = "reconnectAttempts";
+    private static final String DEFAULT_SENDER_TIME_VARIABLE_NAME = "timeStamp";
+    private static final String PREFETCH                          = "prefetch";
```

Comment:
We decided to not use the field alignment any longer (for new code). (https://dev.eclipse.org/mhonarc/lists/hono-dev/msg00305.html)

Reply:
Then I must say, the Hono eclipse settings is not up to date. I will change it there and make it part of this PR.